source/postprocess/gravity_point_values.cc

```diff
@@ -300,35 +486,88 @@ namespace aspect
-          maximum_radius    = prm.get_double ("Maximum radius");
-          minimum_longitude = prm.get_double ("Minimum longitude");
-          maximum_longitude = prm.get_double ("Maximum longitude");
-          minimum_latitude  = prm.get_double ("Minimum latitude");
-          maximum_latitude  = prm.get_double ("Maximum latitude");
+          n_points_radius     = prm.get_double ("Number points radius");
+          n_points_longitude  = prm.get_double ("Number points longitude");
+          n_points_latitude   = prm.get_double ("Number points latitude");
+          minimum_radius      = prm.get_double ("Minimum radius");
+          maximum_radius      = prm.get_double ("Maximum radius");
+          minimum_colongitude = prm.get_double ("Minimum longitude") + 180;
+          maximum_colongitude = prm.get_double ("Maximum longitude") + 180;
+          minimum_colatitude  = prm.get_double ("Minimum latitude") + 90;
+          maximum_colatitude  = prm.get_double ("Maximum latitude") + 90;
+          reference_density   = prm.get_double ("Reference density");
+          radius_list    = Utilities::string_to_double(Utilities::split_string_list(prm.get("List of radius")));
+          longitude_list = Utilities::string_to_double(Utilities::split_string_list(prm.get("List of longitude")));
+          latitude_list  = Utilities::string_to_double(Utilities::split_string_list(prm.get("List of latitude")));
+          AssertThrow (longitude_list.size() == latitude_list.size(),
+                       ExcMessage ("Make sure you have the same number of point coordinates in the list sampling scheme."));
           AssertThrow (dynamic_cast<const GeometryModel::SphericalShell<dim>*> (&this->get_geometry_model()) != nullptr ||
                        dynamic_cast<const GeometryModel::Sphere<dim>*> (&this->get_geometry_model()) != nullptr ||
                        dynamic_cast<const GeometryModel::Chunk<dim>*> (&this->get_geometry_model()) != nullptr,
-                       ExcMessage ("This postprocessor can only be used if the geometry "
-                                   "is a sphere, spherical shell or spherical chunk."));
+                       ExcMessage ("This postprocessor can only be used if the geometry is a sphere, spherical shell or spherical chunk."));
           AssertThrow (! this->get_material_model().is_compressible(),
                        ExcMessage("This postprocessor was only tested for incompressible models so far."));
           AssertThrow (dim==3,
                        ExcMessage("This postprocessor was only tested for 3D models."));
+          output_interval = prm.get_double ("Time between gravity output");
+          if (this->convert_output_to_years())
+            output_interval *= year_in_seconds;
+          maximum_timesteps_between_outputs = prm.get_integer("Time steps between gravity output");
         }
         prm.leave_subsection();
       }
       prm.leave_subsection();
     }
 
+    // IS THIS NEEDED ???
```

Comment:
This is needed if you want to have correct behavior during checkpoint/restart cycles, so yes. Please remove comment. Did you test if the checkpoint/restart works? I.e. does the postprocessor write at the correct time after a restart?